modules/queue/base_redis_test.go

```diff
@@ -56,7 +56,7 @@ func TestBaseRedis(t *testing.T) {
 	}()
 	if !waitRedisReady("redis://127.0.0.1:6379/0", 0) {
 		redisServer = redisServerCmd(t)
-		if redisServer == nil && os.Getenv("CI") != "" {
+		if redisServer == nil || os.Getenv("CI") != "" {
```

Comment:
```suggestion
		if redisServer == nil && os.Getenv("CI") == "" {
```
Isn't this what we want so that if redis fails to load in CI we don't skip the test accidentally?

I agree with running tests locally in general, but I don't think we should force all developers to set up a local redis server.